snippets/csharp/keywords/FixedKeywordExamples.cs

```diff
@@ -39,6 +40,27 @@ unsafe private static void ModifyFixedStorage()
         }
         // </Snippet1>
 
+        // <SnippetFixedSpan>
+        unsafe private static void FixedSpanExample()
+        {
+            int[] PascalsTriangle = {
+                          1,
+                        1,  1,
+                      1,  2,  1,
+                    1,  3,  3,  1,
+                  1,  4,  6,  4,  1,
+                1,  5,  10, 10, 5,  1
+            };
+
+            Span<int> RowSix = new Span<int>(PascalsTriangle, 10, 5);
```

Comment:
It looks to be `RowFive` that starts at element 10 and contains 5 elements.
Or let's update argument values:
```csharp
Span<int> RowSix = new Span<int>(PascalsTriangle, 15, 6);
```

Reply:
Yup, I changed the variable name.